packages/x-archetype-utils/src/i18n/i18n.plugin.ts

```diff
@@ -76,7 +77,7 @@ export class I18n<SomeMessages> {
     // TO-DO Support function or array messages
     const message = path
       .split('.')
-      .reduce<LocaleMessage>(
+      .reduce<any>(
         (messages, key, index, pathParts) =>
           isObject(messages)
```

Comment:
It's just me or are we getting an `Unsafe call of an `any` typed value.` error here?